core/vm/logger.go

```diff
@@ -152,6 +159,9 @@ func (l *StructLogger) CaptureState(env Environment, pc uint64, op OpCode, gas,
 	log := StructLog{pc, op, new(big.Int).Set(gas), cost, mem, stck, storage, env.Depth(), err}
 
 	l.logs = append(l.logs, log)
+	l.sizeLogs += 1
```

Comment:
Too much ruby coding recently (where you cannot increment by `++`).  I will change this.